src/content/en/fundamentals/accessibility/00-intro/intro-three.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: shared/narrow
+title: "Understanding Users' Diversity"
+description: "Accessibility Codelab"
+published_on: 2016-03-01
+updated_on: 2016-03-01
+order: 3
+translation_priority: 0
+authors:
+  - megginkearney
+  - dgash
+key-takeaways:
+  tldr: 
+    - "Learn what accessibility means and how it applies to web development."
+    - "Learn how to make web sites accessible and usable for everyone."
+    - "Learn how to include basic accessibility with minimal development impace."
+    - "Learn what HTML features are available and how to use them to improve accessibility."
+    - "Learn about advanced accessibility techniques for creating polished accessibility experiences."
+notes:
+  efficiency:
+    - "Understanding the accessibility issue, its scope, and its impact can make you a better web developer."
+  problem-solving:
+    - "Catching, identifying, and removing potential accessibility roadblocks before they happen can improve your development process and reduce maintenance requirements."
+---
+
+To explain further, here's an informative question/answer session with Victor Tsaran, a Technical Program Manager at Google, who is totally blind.
+
+>![Victor Tsaran](imgs/victor_tsaran.jpg)&nbsp;&nbsp;&nbsp;**Victor Tsaran**
+
+ - *What do you work on at Google?*
+
+>Here at Google my job is to help ensure that our products work for all of our diverse users, regardless of impairment or ability.
+
+ - *What kinds of impairments do users have?*
+
+>When we think about the kinds of impairments which would make it difficult for someone to access our content, many people will immediately picture a blind user like me. And it's true, this impairment can really make it frustrating or even impossible to use a lot of web sites.
+>
+>A lot of modern web techniques have the unfortunate side effect of creating sites which don't work well with the tools used by blind users to access the web. However, there is actually more to accessibility than that. We find it useful think of impairments falling into four broad buckets: visual, motor, hearing, and cognitive.
+
+ - *Let's go through those one at a time. Can you give some examples of visual impairments?*
```

Comment:
@dgash I sent you an email with a drive folder of images that can be inserted into this interview